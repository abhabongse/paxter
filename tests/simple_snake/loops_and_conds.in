@!##{
    apple = 1 > 2
    orange = 1 < 2
    test = lambda: v < 5
    values = range(10)
}##\
@for[v,values]{\
    @v, @{v + 1}, @{v + 2} |@if[test]{yes}@if[test,false]{no}|
}\
@if[apple]{Apple.}@if[apple,true]{Apple is True.}@if[apple,false]{Apple is False.}
@if[orange]{Orange.}@if[orange,true]{Orange is True.}@if[orange,false]{Orange is False.}
