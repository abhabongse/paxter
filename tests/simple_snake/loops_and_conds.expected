    0, 1, 2 |yes|
    1, 2, 3 |yes|
    2, 3, 4 |yes|
    3, 4, 5 |yes|
    4, 5, 6 |yes|
    5, 6, 7 |no|
    6, 7, 8 |no|
    7, 8, 9 |no|
    8, 9, 10 |no|
    9, 10, 11 |no|
Apple is False.
Orange.Orange is True.
